Cargo.toml

```diff
@@ -26,7 +26,7 @@ aws_lc_rs = ["rustls/aws_lc_rs", "webpki/aws_lc_rs"]
 
 [dependencies]
 # Keep in sync with RUSTLS_CRATE_VERSION in build.rs
-rustls = { version = "0.23.4", default-features = false, features = ["std", "tls12"] }
+rustls = { version = "0.23.11", default-features = false, features = ["std", "tls12"] }
```

Comment:
Commit message says 0.23.12, this says 0.23.11.

Reply:
Good catch, thanks! rebase mixup :-) All fixed.